lib/streamlit/state/session_state.py

```diff
@@ -97,7 +97,7 @@ def __getitem__(self, k: str) -> Any:
                     # No deserializer, which should only happen if state is gotten from a reconnecting browser
```

Comment:
huh, nothing actionable here, but I wonder why this change showed up since if I remember correctly it was one that I made / should have landed on `develop` awhile back 🤷